src/memary/agent/chat_agent.py

```diff
@@ -54,16 +52,6 @@ def clearMemory(self):
             print(f"Error deleting nodes: {e}")
         print("Nodes deleted from Neo4j.")
 
-
-
-    def _add_contexts_to_llm_message(self, role, content, index=None):
```

Comment:
Where did this code move?
Is this the reason persona input doesn't affect the response?

Reply:
I moved that to base_agent.py since one of the methods in that class called it so now chat_agent just inherits the same function

Reply:
I think the output issue was happening a little bit before this too